lib/pluginmanager/remove.rb

```diff
@@ -32,6 +32,14 @@ def execute
     ##
     LogStash::Bundler.setup!({:without => [:build, :development]})
 
+    if LogStash::PluginManager::ALIASES.has_key?(plugin)
+      unless LogStash::PluginManager.installed_plugin?(plugin, gemfile)
+        aliased_plugin = LogStash::PluginManager::ALIASES[plugin]
+        puts "Cannot remove the alias #{plugin}, which is an alias for #{aliased_plugin}; if you wish to remove it, you must remove the aliased plugin instead."
```

Comment:
Another case of variable naming inversion. 

I also adjusted the wording because the pronoun `it` in the original can be read to point to either the alias or the resolved concrete plugin.

```suggestion
        resolved_plugin = LogStash::PluginManager::ALIASES[plugin]
        puts "Cannot remove the alias #{plugin}, which is an alias for #{resolved_plugin}; if you wish to remove #{resolved_plugin}, you can do so directly."
```